src/material/input/input.ts

```diff
@@ -163,6 +164,12 @@ export class MatInput extends _MatInputMixinBase implements MatFormFieldControl<
    */
   @Input() placeholder: string;
 
+  /**
+   * Name of the input.
+   * @docs-private
+   */
+  @Input() name: string;
```

Comment:
Out of curiosity: Do you know why things like `readonly` are not docs-private, but this is?

Reply:
I don't know the context behind it. I saw that `id` and `placeholder` were `docs-private`.

Reply:
I don't have a strong opinion. I guess I'd vote for making them all not `@docs-private` so there's less ugly comments to look at - but thats not a great reason